oxidation/libparsec/crates/client-connection/src/authenticated_cmds.rs

```diff
@@ -0,0 +1,318 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+//! Send Authenticated commands to the server.
+//!
+//! The HTTP-request will contain the following headers:
+//!
+//! ```text
+//! Authorization: PARSEC-SIGN-ED25519
+//! Author: [base64 string]
+//! Timestamp: [timestamp in millisecond]
```

Comment:
what is the format of the timestamp ? just a number or a text format (e.g. `2000-01-01T00:00:00Z`)

Reply:
for me a timestamp is represented as a number.
on the contrary a date would be represented in a text format (following some rfc).

currently the timestamp is formatted like a number (see `timestamp.as_millis().to_string()`).
What would be the preferred format ? number or date string ?

Reply:
I also prefer `ISO8601`

Reply:
done

Reply:
We should use RFC822 instead if ISO8601 (given to be coherent wet the other date-related headers)